#label-> list of features mapping

string 			 person_name
spencer_vision_msgs/PersonEmbeddings person_embeddings
bool			 reset_label